### PR TITLE
[16.0][MIG] partner_bank_code

### PR DESCRIPTION
**Pull request by hbrunn**
I took the liberty to add overrides for name_search, name_get using those fields here

### CODE REVIEW

File: partner_bank_code/models/res_bank.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                            if self.browse(_id).bank_branch_code
+                            else "",
+                        )
+                    )
+                    if self.browse(_id).bank_code
+                    else ""
+                ),
+            )
+            for _id, name in result
+        ]
```
Comment (NL66278) [inline partner_bank_code/models/res_bank.py:44]: I just wondered whether the repeated browse could be prevented with a little helper function, something along the lines off:
```
def make_name(id, name):
    bank = self.browse(id)
    return " ".join(
        [
            name,
            "/".join(
                [
                    part
                    for part in [bank.bank_code, bank.bank_branch_code]
                    if part
                ]
            )
        ]
    )
result = super().name_get()
return [(id, make_name(id, name)) for id, name in result]
```

**Review by NL66278 — APPROVED** [review]
:+1: LGTM Nice addition, one suggestion/question.

File: partner_bank_code/models/res_bank.py (hunk 1 above)
Comment (hbrunn) [reply partner_bank_code/models/res_bank.py:44]: we could, but nowadays this just looks at the same cache, so that's basically syntactic sugar

Comment (NL66278) [reply partner_bank_code/models/res_bank.py:44]: It was  a suggestion. I might be old fashioned in not wanting to retrieve the same data repeatedly, even though I am aware of the caching. But I approved the PR anyway already.

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**